packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -273,22 +270,21 @@ const handleSelect = (key: string) => {
 	}
 };
 
-const onResize = (event: UIEvent) => {
-	void callDebounced(onResizeEnd, { debounceTime: 100 }, event);
-};
-
-const onResizeEnd = async (event: UIEvent) => {
-	const browserWidth = (event.target as Window).outerWidth;
-	await checkWidthAndAdjustSidebar(browserWidth);
-};
+function onResize() {
+	void callDebounced(onResizeEnd, { debounceTime: 100 });
```

Comment:
```suggestion
	void callDebounced(onResizeEnd, { debounceTime: 250 });
```
250 is a bit better, it gives the browser more time and the human eye is not able to tell the difference

Reply:
Good point. Updated